### PR TITLE
docs(slider): Add SliderMarks example

### LINKED ISSUE #78: (docs): `SliderMark` not mentioned in the doc, and `SliderThumb` with tooltip
Hello,

  The `SliderMark` tag is not mentioned in the [online documentation](https://chakra-ui.com/). [Here](https://codesandbox.io/s/test-5lcco?file=/src/App.js) is an example.

  That said, I find it's easier to set a tooltip to the `SliderThumb`.

```jsx
  const [sliderValue, setSliderValue] = React.useState(5);
  <Slider id="slider" defaultValue={5} min={0} max={10} colorScheme="blue" onChange=(v) => {setSliderValue(v)}>
    <SliderTrack>
      <SliderFilledTrack />
    </SliderTrack>
    <Tooltip hasArrow bg="red.600" placement="top" closeOnClick={false} isOpen label={sliderValue}>
      <SliderThumb />
    </Tooltip>
  </Slider>
```

### PR DESCRIPTION
**Pull request by YenTingWu**
Closes #78 

## 📝 Description

Add 2 example to show how to add a mark on `SliderThumb`

- With `SliderMark`
- With `Tooltip`

## ⛳️ Current behavior (updates)

No example for adding marks on thumb

## 🚀 New behavior

- With `SliderMarks`
- 
截圖 2021-12-20 下午1 21 51

- With `Tooltip`
- 
截圖 2021-12-20 下午1 21 57

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by lazarnikolov94 — CHANGES_REQUESTED** [review]
Let's use this for the Tooltip Slider example:
```typescript
function SliderThumbWithTooltip() {
  const [sliderValue, setSliderValue] = React.useState(5);
  const [showTooltip, setShowTooltip] = React.useState(false);
  return (
    <Slider
      id="slider"
      defaultValue={5}
      min={0}
      max={100}
      colorScheme="teal"
      onChange={(v) => setSliderValue(v)}
      onMouseEnter={() => setShowTooltip(true)}
      onMouseLeave={() => setShowTooltip(false)}
    >
      <SliderMark value={25} mt="1" ml="-2.5" fontSize="sm">
        25%
      </SliderMark>
      <SliderMark value={50} mt="1" ml="-2.5" fontSize="sm">
        50%
      </SliderMark>
      <SliderMark value={75} mt="1" ml="-2.5" fontSize="sm">
        75%
      </SliderMark>
      <SliderTrack>
        <SliderFilledTrack />
      </SliderTrack>
      <Tooltip
        hasArrow
        bg="teal.500"
        color="white"
        placement="top"
        closeOnClick={false}
        isOpen={showTooltip}
        label={`${sliderValue}%`}
      >
        <SliderThumb />
      </Tooltip>
    </Slider>
  );
}
```

File: pages/docs/form/slider.mdx [hunk 1]
```diff
@@ -111,6 +114,89 @@ accomplished using the `step` prop.
...
+      </SliderMark>
+      <SliderTrack>
+        <SliderFilledTrack />
+      </SliderTrack>
+      <Tooltip
+        hasArrow
+        bg='teal.500'
+        color='white'
+        placement='top'
+        closeOnClick={false}
```
Comment (nikolovlazar) [inline pages/docs/form/slider.mdx:189]: Actually this prop is redundant as well. Otherwise it's ready to be merged!

**Review by nikolovlazar — CHANGES_REQUESTED** [review]

File: pages/docs/form/slider.mdx (hunk 1 above)
Comment (YenTingWu) [reply pages/docs/form/slider.mdx]: Done!

**Comment by nikolovlazar** [comment]
🚀

**Pull request merged**